### PR TITLE
Bear_Installation_Tool: Remove '--' from all cib commands

### PR DESCRIPTION
**Pull request by alexmstancu**
Fixes https://github.com/coala/documentation/issues/302

### CODE REVIEW

File: Developers/Bear_Installation_Tool.rst [hunk 1]
```diff
@@ -20,39 +20,39 @@ Usage
```
Comment (Nosferatul) [inline Developers/Bear_Installation_Tool.rst]: `Fixes` -> `Closes` in the commit message

Comment (Nosferatul) [inline Developers/Bear_Installation_Tool.rst]: We use `Fixes` for bugs

**Comment by arjunsinghy96** [comment]
Please see http://docs.coala.io/en/stable/Getting_Involved/Writing_Good_Commits.html

File: Developers/Bear_Installation_Tool.rst (hunk 1 above)
Comment (alexmstancu) [inline Developers/Bear_Installation_Tool.rst]: ok, done :D

**Comment by Adrianzatreanu** [comment]
ack 422f5c0cc86fddbb09302660a75e25cc46d4e696

**Comment by Adrianzatreanu** [comment]
cool work there, @alexmstancu . do you mind taking a few mins and filling in this form thatd help us a lot :p ? coala.io/newform . Also, try going for a coala.io/low issue now, thatd be amazing. if you need help picking one up, let us know on gitter!

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Pull request merged**